docs/howto/openstack/nova/resize-server.md

```diff
@@ -0,0 +1,131 @@
+# Resizing a server
+
+This guide will walk you through the required steps to change the number of CPU cores and the amount of memory your server has access to, this is done my changing the server's [flavor](/reference/flavors/).
+
+[Resize](https://docs.openstack.org/nova/latest/admin/configuration/resize.html) (or Server resize) is the ability to change the flavor of a server, thus allowing it to upscale or downscale according to user needs. A resize operation is a two-step process for the user:
+
+1. Initiate the resize.
+2. Either confirm (verify) success and release the old server, or
+   declare a revert to release the new server and restart the old one.
+
+## Prerequisites
+
+You need to have a server you wish to resize. Additionally, if you prefer to work with the OpenStack CLI, then make sure to properly [enable it first](/howto/getting-started/enable-openstack-cli/).
+
+## Listing available flavors
+
+=== "{{gui}}"
+    Navigate to the server list.
+
+    ![The left hand side navigation panel, with the word "Servers" highlighted.](assets/resize-server/01-left-side-panel.png)
+
+    Find the server you want to resize in the list and on the right hand side click on it's menu button.
```

Comment:
* Insert comma before "and".
* Use "right-hand side".
* Replace "it's" with "its".